tools/ITER_Profile_reader.py

```diff
@@ -0,0 +1,66 @@
+# This class provides routines for reading input profiles from CORSICA data files
+
+import numpy as np
+
+class ITER_Profile_reader:
```

Comment:
If this script loads CORSICA profiles, wouldn't a more natural name be ``CORSICA_Profile_reader``? (Or even ``CORSICAProfileReader``?)